spec/models/activity_pub/actor_spec.cr

```diff
@@ -166,7 +171,12 @@ Spectator.describe ActivityPub::Actor do
           "mediaType": "image/jpeg",
           "url": "image link"
         },
-        "url":"url link"
+        "url":"url link",
```

Comment:
the `@context` section in this json-ld document needs to be changed to:
```
        "@context":[
          "https://www.w3.org/ns/activitystreams",
          "https://w3id.org/security/v1",
          {
            "schema":"http://schema.org#",
            "PropertyValue":"schema:PropertyValue",
            "value":"schema:value"
          }
        ],
...
```
this will ensure that when the json-ld is expanded, the attachment type and value are expanded correctly. name is already defined by the activity streams core schema.

Reply:
Great, thanks! Will try to get it cleaned up this weekend